releasenotes/notes/migrate-to-qiskit-nature-0.5.0-96e90cd48f36d731.yaml

```
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    Full support for Qiskit Nature 0.5.0.
+deprecations:
+  - |
+    Support for Qiskit Nature < 0.5.0 will be removed upon this release.
+
+    Users will now be required to use the ``:class:~qiskit_nature.second_q.problems.ElectronicStructureProblem``,
```

1. I think this is actually an "upgrade" release note.  It would fall under "deprecations" if we had deprecated but not removed support for earlier versions of Nature.
2. I think the release notes should be in the present tense.  If we'd prefer that notes from the _upcoming_ release not appear on our website, then we should adjust Sphinx to build docs from the latest stable version rather than `main`.  (Alternatively, we might be able to find a way for it to disable upcoming-release notes within the build on CI.)

```suggestion
upgrade:
  - |
    Support for Qiskit Nature < 0.5.0 is removed with this release.

    Users are now required to use the ``:class:~qiskit_nature.second_q.problems.ElectronicStructureProblem``,
```